scripts/new-validation.py

```diff
@@ -0,0 +1,73 @@
+from pathlib import Path
+
+import pandas as pd
+import pandera as pa
+
+from tables.schema import TableSchema
+
+resources = Path(__file__).parent / "resources"
+
+
+def run_new_validation():
+    """Provides an example of using a TableSchema to extract and validate a table from a spreadsheet.
+
+    The spreadsheet can be found at "scripts/resources/new-validation-example-spreadsheet.xlsx".
+        - Modify the data in the spreadsheet so that it violates the schema and re-run to see ErrorMessages.
+        - Modify the TableSchema to extract a different subset of columns from the data.
+    """
+
+    scores = ["1 - very low", "2 - low", "3 - medium", "4 - high", "5 - very high"]
+    risk_categories = ["Funding", "Procurement", "Staff"]
+
+    risk_schema = TableSchema(
+        id_tag="TABLE001",
+        worksheet_name="Risks",
+        section="Risks",
+        columns={
+            "Risk name": pa.Column(str, unique=True, report_duplicates="exclude_first"),
+            "Category": pa.Column(str, pa.Check.isin(risk_categories)),
+            "Description": pa.Column(str),
+            "Likelihood score": pa.Column(str, pa.Check.isin(scores)),
+            "Impact score": pa.Column(str, pa.Check.isin(scores)),
+            "Total risk score": pa.Column(int),  # Comment this out and this column won't be extracted and validated
+            "Mitigations": pa.Column(str),
+        },
+        row_idxs_to_drop=[0],  # Drop the first row with help text
+        drop_empty_rows=True,
+        # num_dropped_columns=1,  # Uncomment this if you remove "Total risk score" from the columns
+        messages={
+            "isin": "You’ve entered your own content, instead of selecting from the dropdown list provided. "
```

Comment:
Not sure if ```isin``` is the most intuitive key. Something like ```enum_failure``` (which I think this always relates to?) might be better for readability.